web/cobrands/sass/_admin.scss

```diff
@@ -343,6 +343,15 @@ textarea.code {
     max-width: initial;
 }
 
+// Shrink the sidebar to make more room for the main content
+.container--sidebar .sticky-sidebar {
```

Comment:
Don't follow this fixup, sorry - this changes the sidebar everywhere, rather than just the bulky items page? Might have misunderstood.

Reply:
Yep, that's right. Figured it didn't hurt and made a little more room for main content on all admin pages. Do you think that's OK?

Reply:
It's not just the admin though - it's all pages :)

Reply:
Oops, I thought `_admin.scss` was scoped to admin pages. Will update.